common/errno_mapping.cc

```diff
@@ -166,4 +166,12 @@ absl::Status ErrnoToCanonicalStatus(int error_number,
                       absl::StrCat(message, ": ", strerror(error_number)));
 }
 
+absl::Status ErrorCodeToCanonicalStatus(const std::error_code& code,
+                                        absl::string_view message) {
```

Comment:
If you change the function signature to accept string formatting args, similar to how we do it for [WrapStatus()](https://github.com/google/cdc-file-transfer/blob/main/common/status.h#L50), you can simplify the calls to this function.

Reply:
Done.